.travis.yml

```diff
@@ -51,50 +55,8 @@ jobs:
     - name: plugin install
       script: ./gradlew install
```

Comment:
This is an existing problem, but I think it might be worth tweaking this step so that it's combined with the unit test job, as it'll skip 9 minutes of what is mostly environment setup on CI.